pypeit/core/flux_calib.py

```diff
@@ -1012,43 +1026,25 @@ def get_mask(wave_star, flux_star, ivar_star, mask_star,
     atms_cutoff = wave_star <= 3000.0
     mask_bad[atms_cutoff] = False
 
-    # TODO JFH replace with mask_star_lines from telluric.py
-    if mask_abs_lines:
-        # Mask Balmer, Paschen, Brackett, and Pfund recombination lines
-        msgs.info("Masking recombination lines:")
-        # Mask Balmer
-        msgs.info(" Masking Balmer")
-        lines_balm = np.array([3836.4, 3969.6, 3890.1, 4102.8, 4102.8, 4341.6, 4862.7, 5407.0,
-                               6564.6, 8224.8, 8239.2])
-        for line_balm in lines_balm:
-            ibalm = np.abs(wave_star - line_balm) <= balm_mask_wid
-            mask_balm[ibalm] = False
-        # Mask Paschen
-        msgs.info(" Masking Paschen")
-        # air wavelengths from:
-        # https://www.subarutelescope.org/Science/Resources/lines/hi.html
-        lines_pasc = np.array([8203.6, 8440.3, 8469.6, 8504.8, 8547.7, 8600.8, 8667.4, 8752.9,
-                               8865.2, 9017.4, 9229.0, 9546.0, 10049.4, 10938.1,
-                               12818.1, 18751.0])
-        for line_pasc in lines_pasc:
-            ipasc = np.abs(wave_star - line_pasc) <= balm_mask_wid
-            mask_balm[ipasc] = False
-        # Mask Brackett
-        msgs.info(" Masking Brackett")
-        # air wavelengths from:
-        # https://www.subarutelescope.org/Science/Resources/lines/hi.html
-        lines_brac = np.array([14584.0, 18174.0, 19446.0, 21655.0,26252.0, 40512.0])
-        for line_brac in lines_brac:
-            ibrac = np.abs(wave_star - line_brac) <= balm_mask_wid
-            mask_balm[ibrac] = False
-        # Mask Pfund
-        msgs.info(" Masking Pfund")
-        # air wavelengths from:
-        # https://www.subarutelescope.org/Science/Resources/lines/hi.html
-        lines_pfund = np.array([22788.0, 32961.0, 37395.0, 46525.0,74578.0])
-        for line_pfund in lines_pfund:
-            ipfund = np.abs(wave_star - line_pfund) <= balm_mask_wid
-            mask_balm[ipfund] = False
+    if mask_hydrogen_lines:
+        mask_recomb = mask_stellar_hydrogen(
+            wave_star, mask_width=balm_mask_wid, mask_star=mask_recomb
+        )
+
+    if mask_helium_lines:
```

Comment:
Should we create a mask helium lines function similar to what you did for hydrogen?

Reply:
Done.